streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -711,6 +715,56 @@ public void shouldUseSourceSpecificDeserializers() {
         assertThat(result2.getValue(), equalTo(source2Value));
     }
 
+    @Test
+    public void shouldPassRecordHeadersIntoDeSerializers() {
```

Comment:
DeSerializers -> Deserializers

Reply:
The capital `S` was on purpose; it's short for `IntoDeserializersAndSerializers`